discovery-provider/src/api/v1/tracks.py

```diff
@@ -131,6 +131,10 @@ class Trending(Resource):
     @record_metrics
     @ns.doc(
         id="""Trending Tracks""",
+        params={
+            'genre': 'Trending tracks for a specified genre',
```

Comment:
Is there any way to mark these as optional? Or are they optional by default?

Reply:
optional by default! or rather, "required = false"